hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -23,7 +23,10 @@
   build_pass_patterns: []
   // TODO: Add back FuseSoC fail pattern after
   // https://github.com/lowRISC/opentitan/issues/7348 is resolved.
-  build_fail_patterns: []
+  // Note(#10083, #10952): ENUMASSIGN issues can cause LEC warnings 
+  // later down the road. The warning is therefore promoted to an error 
+  // in simulations in order to catch this as early as possible.
+  build_fail_patterns: ["^Warning-\\[ENUMASSIGN\\].*"] 
```

Comment:
this warning pattern may be VCS specific. If so, we should add it in dvsim/vcs.hjson and add another one in dvsim/xcelium.hjson

Reply:
Oh that's a good point. Let me move it over then.